src/ext/ddtrace.h

```diff
@@ -16,5 +16,6 @@ ZEND_END_MODULE_GLOBALS(ddtrace)
sS4MGZdAGfFd4/QSRQPI9K6IKEPF4j2JhbMutrBXD8xXOVhacUnK2HDNgEnU4DeZznxwPGtAxd8uLBjRcG+kIkFk6EJemokU8X4lv8mrPgSwszd+dcpTjy4AnutGucRdqyIAt3sEgsmyXEIkxBzM4/9l6UV34bWDXYLLgVXcTta1qag9l10m6EmFswqexISczPPPRh/LGYTadftIscUFUpibZKWTPPnvISdQwtW2UgsmDye9jDyFguAhVhMiqz83HH7dauKKO4Q6PNRrTcOtRIJrEuQiOl/4XMkFoxLrrGYlYgFyeIN6+Kgapeh16/7hSD+2182pV23i0msi4Pa6ffHqrQEk2Ms5ooF/bEKsQDL8QbdeTxPCrhzdHVgpveetXn2UyZ2jENYyv/xx6qS7Va75BSLcVIXv65mygOc5SP50hx0KmHniZOInmmKfCGI/2amxDNBcNmsSlHVszDmYzGXNFt9Nlpw+Riya7tyVI6uJir/Z05w2RyG//poCcZwLObLzTZV1rGxGVwNBHdt10FCRzd3wvw7//XRmpJMNUEn4MztnrqAtXd6DEmRoWw48+uEwaMsWW3B7LJ1Fxt0HF3WcXTNEUiSIsFTDs5JvkQzLcGYIG4lFJYuae3ZFcmyLJjqrpBeEOi1ez7oCxF1U57zmVelWHfN4AK4m/0xTEeiDDxfopnWlJTyxsxJs2zufGuOPg73u53zVoVm9BsRf8Dd8v41M/91VOsNdNMUFpKFQlIRV4UBR9csis1ZLcFkIY1YgnS+Nyedb/vHXjth30uvn91yeAArAv9cnegpi+GgapePar3pUa03sKqnjURhexh3dFOzkCQVko7rF7S2YNLk1poQi5/uoDntDFsNFvQHXGvDwHNBPEra9CIYi0l6k8MQxH04DutrJrK5wP8c1Xpja+/0OE7Em+Lo6qAtGF0Y/M6kWPx8/NocSKYK7qaoHSayk4km0Dl3K9205EZYQ89iOzXs+O+jWm9q1Xr9w1d23Zs6U0R0c2OhAGLEbr73QOkLRqPxFjEuTFePCvLpvDmmQGcUJrJ1picgcNESclC1ywnrGO8w8IYk/+/pLf+fVe2NXKu0EfgLIEauPN0HSl8wGqyq0ULnW3PkOsNzBPEozhEOOvA6Xck8Uofx3dWeatwmkkIwyZqgE3C2yghu59v+ccC/2nGLKiYiGCJXcVg7bW/U6iYLvjgLEJX+4WxSphBMQmaLh6ZWQqC/MxN+j8sZ8QssSYjcw9q1S4Q1/H55ETzQF1N8QVswCWMxl2mjsFnofG9OglMTgY6TxGgSJUR7iPmq6NfBtULagkkCUfKpwDTXW6KLxbydncjt/kUHPpFP8aCmIpdgVl3oKVPO4PSqYzHJ/Jw86A6aU6JFZ5SZ2kmsjIoHNxXdsWgtQ5qRehY4lWBUrDL/N4wwK/PkVjaWRy4KO3gsdIlfcSq64wWQVjCKWExhzYLpDppTCpTjICxXXdIR9uEru/6ApqJLAs4Y/I6IXt5s038WXo3ZT8pnt9pXqGZdzJi6gtif1/oiWDny03lzvFhEJ7zwYrtuF+l2cwJtGqSr+yvEBCFpLAdVu5xKMMEUx6XXnSDYKHLACvh03hxbtd5CMUcBblh79kCyLAtQGYxSkrSCp7f3Yiq6BDBxsiLFSKMtYGIKAsVUghGMadyFdnsepYqCmoQZg4X9GsJbZn5LILDz/SIhh9oOX9l1SE58enAVEOOCgTGBxxBikjDUEYm1a5ecWUFWwFRi5tCYlKSUggktoOeDgeeHtdN23vtIKrhAI5LpM9nWPRWRU+52TMRjSWIsbjHOajWsPbsCKUsMKhNQZkKZwTvhT9AigmU5fkQMSRpvmU5rSEOSz+nj6mTYmi+/826S6ocYF/OiiiTGWUvJ+62Gm4JaQsJYUxRE/CGVhXFRNt5yUw2wTtEQ40JjdbPop+RTDDKdIxpDFquhAzOlc3oBKM+veDCRfVTtlU/OW5F7OnnCy723nRtGmMBpXVeGr8LBwkoqYrWgwYIjmpfVYGaAnJuR6dMqIE7pwwChNyIawlur1iuGnRTIm2AWPBHVO8M7J/Gwdtom0Fww/kNtM4mpSPBQwBWhUUfUTX4nppJrNZwG5IathhaEUmrBhB5HiIGBN1ath1WLRhUCCB6d8YcEluM05h1Ra9cuyW2U5237gDIDz5kZBAIoX6uhAwPPUwtGdSPCYKcv9XiVqydVCCBY7HmpFClw5jmieVkNkoDXIEPvihrHsZSE6d0pV2f7xPvdUwtGeSMiINDnw1f2xGjduhgkoUgxH3SpvQ9TaeH1FBZxU60GuX0VPP9NMqaCxDjYWyGO1IJRxWKCkK8JhJjJhac4TwTL8ry9TTR3K75gMlEMTps+lBcix+u1GpfkPMhjEjyVJMaCMTUZ9U0tmLBmBn5RRHUGWTkBixFWZWlhxRdxqM3fIhiSKiCU+JafM4DIyLFh5mkl881fZ7rIuvrSIbVgAIDB7yDERDCmWef3HKn4vwm9sL4GmQw8nx/P9VkNgAHPauQjjAX/wZsugMRZjishk2BW6bym4aBqlxn83Ps+KvEr2EeJmf8ybTWC/gMkJhBiYnK6WAWZBLPpFIjbfv+BI0qVmahK4Ykx6D+scrpYBQ9WME4VyYV8GHCBRmFjEwToFrvIOl+MgM2aLlbBgxWMm065wL+F8BydT+fNsVXtXQT9h7U77BuIoRl6s3DD/Z/9/0YRRYwe0SNdTm9OGMnsr542gmIBABD1l0c/ostGTUnPfsr2UbVXvHlCx7qOYrtuF5/ecp/Da+t/+dV8jbzYuCnpqNYbAygRMJgxdVU+hK9FcAPhebdXN9tU0hXgI+FsnGDadbv47JbHfJe8fZdw5McpnlhGfHK2UwP40XE1xsYJBnCXxAUeIFtK4aVkajyKxSwbKRhgbmm6nLhm/pwrIu4maWD+iD4bKxgPa9cuseA2CBVEW5xLAsZS0GBVaRO/Kv8POMFDR3RHrcUAAAAASUVORK5CYII="
```

Comment:
I'm always happy to see Bits. But I'm concerned with how many bits he takes up here :) 

I would love to see Bits once we're officially out of Beta. For now I think it would be best to avoid adding strings of over 9000 bytes length. 

Sounds good?